docker-extension-annotations/src/docker/annotation.bal

```diff
@@ -49,6 +50,7 @@ public type DockerConfiguration record {|
     string dockerHost?;
     string dockerCertPath?;
     map<boolean|int|float|decimal|string> env?;
+    string dockerConfigFile?;
```

Comment:
Shall we change the field name to `dockerConfigPath`. Similar to `dockerCertPath`.